src/filter-functions/androidHandler.js

```diff
@@ -0,0 +1,14 @@
+// export const androidHandler=(data,state)=>{
+//     let arr=[]
+//     if(!state.category.android)
+//     {
+//         return data
+//     }
+//     if(state.category.android)
+//     {
+//         arr=[...data].filter(item=>item.android)
+//         return arr
+//     }else{
+//         return arr
+//     }
```

Comment:
if this code is not required then better to delete this file.

Reply:
yes got it